inst/css/filter-panel.css

```diff
@@ -90,15 +106,29 @@ a.remove_all:hover {
   color: #d40000;
 }
 
-.filterPlotOverlayBoxes {
+.choices_state label {
+  display: flex;
+  gap: 5px;
+}
+
+.choices_state label > input {
+  flex-grow: 0;
+}
+
+.choices_state label > span {
+  flex-grow: 4;
+}
+
+.choices_state_label_text {
   position: absolute;
-  top: -2px;
-  right: 16px;
-  bottom: -2px;
-  left: 16px;
-  animation: 0.75s ease-out 0s 1 shinyDataFilterEnlargeX,
-    0.5s ease-in 0s 1 shinyDataFilterFadeIn;
-  transform-origin: left;
+  padding: 2px;
+}
+
+.choices_state_label {
+  background-color:lightblue;
```

Comment:
Linking this to insightsengineering/teal/issues/735. If we match the whole app theme to the user set bootstrap theme, we will want to update the color here.